pythonx/jedi_vim.py

```diff
@@ -25,6 +25,15 @@
     ELLIPSIS = u"…"
 
 
+try:
+    sys.prefix = sys.base_prefix
+    sys.exec_prefix = sys.base_exec_prefix
+except AttributeError:
+    # Somehow sys.prefix is set in combination with VIM and virtualenvs.
+    # However the sys path is not affected. Just reset it to the normal value.
+    pass
```

Comment:
What is this needed for?  (in the case of not AttributeError)

Reply:
Bad documentation. Sorry. It's at least somewhat better, now.